sources/dicom/test_dicom/web_client_specs/dicomWebSpec.js

```diff
@@ -70,15 +70,15 @@ describe('DICOMWeb assetstore', function () {
                 type: 'GET',
                 async: false,
                 data: {
-                    parentType: 'user',
-                    parentId: userId,
+                    destinationType: 'user',
```

Comment:
These need to still be `parentType` and `parentId`, since it is the folder endpoint and that hasn't changed.